### PR TITLE
[DEV-3362] Reward Exchange visual-dev updates

### PR DESCRIPTION
**Pull request by noahwc**
## Description of the change

Visual dev changes for the self-serve reward exchange project

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: [DEV-3362](https://saasquatch.atlassian.net/browse/DEV-3362)
- Process.st launch checklist: https://app.process.st/runs/Deployment-Launch-Plan-441PM-workflow-run-jiHUhEkGTqAvFrQ3mPFFHg/tasks/ipJPSwLUytFe3xMt6P5PYw

### Development

- [x] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [] The behaviour changes in the pull request are covered by specs
- [x] All tests related to the changed code pass in development

### Paperwork

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has a Jira number
- [x] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

[DEV-3362]: https://saasquatch.atlassian.net/browse/DEV-3362?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/visual-dev/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
```
Comment (DerekSiemens) [inline packages/visual-dev/CHANGELOG.md:9]: Flagging change log needs updating

**Comment by DerekSiemens** [comment]
Flagging PR body needs updating

File: packages/visual-dev/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+## [1.3.0] - 2023-1-16
```
Comment (DerekSiemens) [inline packages/visual-dev/CHANGELOG.md:10]: Changelog version does not match what is currently in the package.json @noahwc just flagging to make sure everything is lining up before launch

File: packages/visual-dev/src/components/RadioGeneric/RadioGeneric.stories.tsx [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
+// }: {
+//   value: string;
+//   activeValue: string;
+// }) => {
+//   return (
+//     <div style={{ backgroundColor: value == activeValue ? "red" : "blue" }}>
+//       {value}
+//     </div>
+//   );
+// };
```
Comment (tdgao) [inline packages/visual-dev/src/components/RadioGeneric/RadioGeneric.stories.tsx:22]: Flagging commented code

File: packages/visual-dev/src/components/Select/MultiSelect.stories.tsx [hunk 4]
```diff
@@ -342,3 +467,133 @@ export function MultiSelectWithManyItems() {
...
+            }}
+          >
+            {itemToString(selectedItemForRender)}
+          </TagView>
+        );
+      })}
+    </>
+  );
+
+  // const emptySlot = <>EMPTY</>;
```
Comment (tdgao) [inline packages/visual-dev/src/components/Select/MultiSelect.stories.tsx:574]: Flag commented code

File: packages/visual-dev/src/components/Select/MultiSelect.stories.tsx [hunk 5]
```diff
@@ -342,3 +467,133 @@ export function MultiSelectWithManyItems() {
...
+    itemToNode,
+    itemToString,
+  };
+
+  return (
+    <SelectView.ContainerView>
+      <SelectView.HandleView {...props} />
+      <SelectView.ListView
+        {...props}
+        // emptySlot={props.emptySlot}
```
Comment (tdgao) [inline packages/visual-dev/src/components/Select/MultiSelect.stories.tsx:594]: Commented code

File: packages/visual-dev/src/components/Select/MultiSelect.stories.tsx [hunk 6]
```diff
@@ -342,3 +467,133 @@ export function MultiSelectWithManyItems() {
...
+  { title: "1984", author: "George Orwell" },
+];
+
+export function MultipleSelectObjectItem() {
+  const itemToString = (item: Book | null) => {
+    return item?.title || "";
+  };
+  const itemToNode = (item: Book | null) => {
+    if (item === null) {
+      return <></>;
```
Comment (tdgao) [inline packages/visual-dev/src/components/Select/MultiSelect.stories.tsx:489]: Prefer return null

File: packages/visual-dev/src/components/RadioGeneric/RadioGeneric.stories.tsx (hunk 3 above)
Comment (noahwc) [reply packages/visual-dev/src/components/RadioGeneric/RadioGeneric.stories.tsx:22]: This story needs to be fixed at some point, added a todo

File: packages/visual-dev/CHANGELOG.md [hunk 7]
```diff
@@ -220,6 +234,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - \<Tooltip>
 
 [unreleased]: https://github.com/saasquatch/program-tools/compare/visual-dev%401.2.1...HEAD
```
Comment (DerekSiemens) [inline packages/visual-dev/CHANGELOG.md:236]:
```suggestion
[unreleased]: https://github.com/saasquatch/program-tools/compare/visual-dev%401.3.0...HEAD
```

**Review by DerekSiemens — APPROVED** [review]

**Review by tdgao — APPROVED** [review]

**Pull request merged**